_data/developer.yml

```diff
@@ -14,7 +14,7 @@ websites:
       - phone
       - proprietary
       - hardware
```

Comment:
```suggestion
      - u2f
```

Reply:
I see no mention of hardware-based 2FA (U2F or otherwise) - am I missing something?

Reply:
I believe that since all 2FA for Aha is done via Duo, users can use security keys through that proprietary method as well as phone, sms, push notifications. See https://guide.duo.com/security-keys